CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Next release
 
+## 0.35.1 / 2020-01-13
+
+* [ENHANCEMENT] Restrict api extension RBAC rules (#2981)
```

Comment:
hmm did we merge this into 0.35 branch? 🤔 

In patch release/z-stream we don't want ENHANCEMENT or feature work.

Reply:
It's a permission fix, so it's not really a new feature.  I wasn't sure whether to call it a bugfix or enhancement, and decided on enhancement since I didn't see a specific bug referenced.  But probably @s-urbaniak has more info.

Reply:
I would say its CHANGE then? 🤷‍♀ :)

Reply:
Updated, thanks!